Msgbox.go

```diff
@@ -114,22 +114,41 @@ func PrepareMsgbox() Layout {
 	}
 }
 
-func Msgbox(title, content string) {
-	MsgboxV(title, content, MsgboxButtonsOk, nil)
-}
+type MsgboxWidget struct{}
```

Comment:
not sure if this type is necessary.
my second idea is to return MsgboxState (in Msgbox() method) but now it looks better for me.